ui/src/components/admin/DataDictionary/DependentFieldValuesGroup.js

```diff
@@ -61,7 +65,7 @@ const DependentFieldValuesGroup = ({ fieldName, fieldKey, fieldValues, isOpen =
                   setNewFieldValue(e.target.value);
```

Comment:
should we do the `trim()` here instead of when the value is used?

Reply:
When I tried that you weren't able to add fields with spaces in them (multiple words) since it would trim any spaces after you started typing. So if you wanted to add "Some Field" it would just be "SomeField"